Source/DafnyLanguageServer/Plugins/QuickFix.cs

```diff
@@ -0,0 +1,41 @@
+﻿using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.Dafny.LanguageServer.Plugins;
+
+/// <summary>
+/// Describe a single suggestion made to the user, where edits can be
+/// computed lazily.
+/// If the edits can be provided instantly, you can use the derived class
+/// `new InstantQuickFix(title, edits)`
+/// </summary>
+public abstract class QuickFix {
```

Comment:
Could this name be more connected to the name `CodeAction`, to stay in line with LSP's terminology? I believe LSP uses the term `quickfix` to indicate a code action that resolves a diagnostic, but that's not necessarily the case here.

Reply:
> Could this name be more connected to the name CodeAction,

I renamed it to DafnyCodeAction / DafnyCodeActionProvider